lib/oli/resources/collaboration.ex

```diff
@@ -357,30 +357,29 @@ defmodule Oli.Resources.Collaboration do
       from(
         post in Post,
         join: sr in SectionResource,
-        on: sr.resource_id == post.resource_id and sr.section_id == post.section_id,
+        on:
+          sr.resource_id == post.resource_id and sr.section_id == post.section_id and
+            sr.numbering_level > 0,
```

Comment:
NIT: Once we allow posts at the course level, we might want to change this to be `sr.numbering_level == 0`. This is just something to keep in mind. We don't need to change it now 😁